src/op_handlers/div.rs

```diff
@@ -0,0 +1,8 @@
+use crate::address_operands::address_operands_read;
+use crate::{opcode::Opcode, state::VMState};
+
+pub fn _div(vm: &mut VMState, opcode: Opcode) {
+    let (src0, src1) = address_operands_read(vm, &opcode);
+    vm.set_register(opcode.dst0_index, src0 / src1);
+    vm.set_register(opcode.dst1_index, src0 % src1);
```

Comment:
Here you should use address_operands_store for storing the results, right now it only cares about the dst0, it shouldn't be much different to implement it for dst1

Reply:
Right, the logic of both `mul` and `div` for `address_operand_store` need to be implemented then.